tests/test_multiannotator.py

```diff
@@ -6,43 +6,48 @@
 from cleanlab import count
```

Comment:
Do we have tests for the following edge cases:

1. There is only a single labeled example to start with (ok if code crashes just provide informative error msg)
2. No labeled example to start with (ok if code crashes just provide informative error msg)
3. Only a single unlabeled example
4. In the starting labeled dataset, every example has only been labeled by a single annotator

Reply:
Some checks existed + added a few new ones to cover the ones that were missed. 
Expected behavior:
1. Code works (probably really bad results, but code will run)
2. Error message will be printed if any example contains no labels, having no labeled examples at all falls within this error catching
3. Code works
4. `get_label_quality_multiannotator` will throw an error saying 

     > Each example only has one label, collapse the labels into a 1-D array and use cleanlab.rank.get_label_quality_scores instead

Reply:
For 4, can we handle this special case inside `get_active_learning_scores()` so it does not error?  That actually seems like a common scenario, ie. every example has exactly one label to begin with and we want to consider collecting more labels for these vs collecting labels for additional unlabeled examples.

Reply:
^ To be clear I mean just have a special `if` check to see if we are in this scenario and if so, just internally call `cleanlab.rank.get_label_quality_scores()` to get the active learning scores for the labeled examples (and penalize the resulting scores appropriately relative to the scores for the unlabeled data).

Reply:
Let's also have a unit test that does a couple iterations of active-learning (with tiny dataset) in the standard fashion  where every example only gets one label at most. So in each iteration, just select the unlabeled example with the lowest active learning score currently and give it a label. No need to even retrain classifier for the unit test.  I just want to verify this code still generalizes to the traditional active learning setting.